toolbar.less

```diff
@@ -14,7 +14,15 @@
 
     opacity: 0.9;
     background-color: @dark-charcoal;
-
+    .emulate-user-title {
```

Comment:
@ajnassar why? is this separate from the bootstrap stuff?

Reply:
Yes